src/components/Card.js

```diff
@@ -4,17 +4,19 @@ import PropTypes from 'prop-types'
 Card.propTypes = {
   galleries: PropTypes.arrayOf(
     PropTypes.shape({
+      id: PropTypes.string,
       image: PropTypes.node,
       name: PropTypes.string,
       style: PropTypes.string,
       opening: PropTypes.string,
     })
   ),
```

Comment:
You get `galleries` here as prop, you only get the destructured elements. Also, you don't get the `id` here, so you can remove it. Here is how it would look like:
```suggestion
   image: PropTypes.node,
   name: PropTypes.string,
   style: PropTypes.string,
   opening: PropTypes.string,
```

Reply:
Alrighty, done